backend/modules/data/domain/value_objects.py

```diff
@@ -0,0 +1,15 @@
+import pandera as pa
+from pandera.typing import Series
+
+class EtlEntryData(pa.SchemaModel):
+    name: Series[str]
+    surname: Series[str]
+    age: Series[int] = pa.Field(ge=0)
+    is_satisfied: Series[bool]
+
+class EtlOutputData(pa.SchemaModel):
+    name_surname: Series[str]
+    age: Series[int] = pa.Field(ge=0)
```

Comment:
Please notice, that such validator raises exception. Such exception should be caught and reraised with domain exception.